examples/react-js/README.md

```diff
@@ -0,0 +1,20 @@
+## Builder.io example with React
+
+### Quick start
+
+[Open this example in Codesandbox](https://codesandbox.io/s/github/BuilderIO/builder/tree/main/examples/react-js)
+
+<a target="_blank" href="https://codesandbox.io/s/github/BuilderIO/builder/tree/main/examples/react-js">
+  <img width="597" height="375" src="https://i.imgur.com/zue72Q0.jpg">
+</a>
+
+### To run the example Locally
+
+```bash
+git clone https://github.com/BuilderIO/builder.git
+cd examples/react-js
+npm install
+npm run dev
+```
+
+See src/main.jsx for the source
```

Comment:
We need better instructions than that for new examples. I know you copied the one for the react one, but this is an old one and we want more details for our users.

Some examples:
- https://github.com/BuilderIO/builder/blob/main/examples/svelte/sveltekit/README.md
- https://github.com/BuilderIO/builder/blob/main/examples/next-js-simple/README.md


You can probably copy/paste a bunch of the stuff from the READMEs and replace command names.